feat: adding support for --profile cmd option

axel7083 opened the pull request:
#### What type of PR is this?

<!--
Add one of the following kinds:
/kind bug
/kind cleanup
/kind documentation
-->
/kind feature

#### What this PR does / why we need it:

When using docker compose we can use the [profiles](https://docs.docker.com/compose/compose-file/05-services/#profiles) attribute. 

> profiles defines a list of named profiles for the service to be enabled under. If unassigned, the service is always started but if assigned, it is only started if the profile is activated.

In the current behaviour of kompose, if the compose file is using profiles, it will not process the services. This PR allows to add the `--profile` cmd command (repeatable) to send a list of string to the loader, that will be able to process the file and enable the services.

#### Which issue(s) this PR fixes:
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
Fixes https://github.com/kubernetes/kompose/issues/1708 https://github.com/kubernetes/kompose/issues/1458 https://github.com/kubernetes/kompose/issues/1448

#### Special notes for your reviewer:

cdrage commented:
The code LGTM and implementation! But you'll have to add a test if that's okay for it to be all merged in:)

axel7083 commented:
> The code LGTM and implementation! But you'll have to add a test if that's okay for it to be all merged in :)

I've added a test in the client. To do so, I added an example of docker-compose with some profile, and using the tool, I ensure it matches the expected output.

cdrage commented on cmd/convert.go:
```diff
@@ -199,6 +201,8 @@ func init() {
 
 	convertCmd.Flags().IntVar(&ConvertYAMLIndent, "indent", 2, "Spaces length to indent generated yaml files")
 
+	convertCmd.Flags().StringArrayVar(&ConvertProfiles, "profile", []string{}, `Specify the profile to use, can use multiple.`)
```
```suggestion
	convertCmd.Flags().StringArrayVar(&ConvertProfiles, "profile", []string{}, `Specify the profile to use, can use multiple profiles`)
```

cdrage reviewed (changes requested):
One TINNYYYY change (no period at end of sentence and little bit of grammar change), but otherwise LGTM!

cdrage commented:
/approve

cdrage commented:
/lgtm
